src/main/java/fi/hsl/transitdata/gtfsbundler/DatasetBundler.java

```diff
@@ -15,13 +13,30 @@
 
     final HashMap<Long, DatasetEntry> cache = new HashMap<>();
     final int maxAgeInMs;
-    final String path;
     final String fileName;
 
-    public DatasetBundler(Config config) {
+    final ISink sink;
+
+    public enum Destination {
+        local, azure
+    }
+
+    private DatasetBundler(Config config, ISink sink) {
         maxAgeInMs = config.getInt("bundler.maxAgeInMinutes") * 60 * 1000;
```

Comment:
This could be changed to use the duration parsing of the config library

Reply:
good point, fixed